lib/iris/fileformats/netcdf.py

```diff
@@ -2685,6 +2685,28 @@ def add_ellipsoid(ellipsoid):
                     cf_var_grid.false_easting = cs.false_easting
                     cf_var_grid.false_northing = cs.false_northing
 
+                # polar stereo (have to do this before Stereographic because it subclasses it)
+                elif isinstance(cs, iris.coord_systems.PolarStereographic):
+                    if cs.ellipsoid:
+                        add_ellipsoid(cs.ellipsoid)
+                    cf_var_grid.latitude_of_projection_origin = cs.central_lat
+                    cf_var_grid.straight_vertical_longitude_from_pole = (
+                        cs.central_lon
+                    )
+                    cf_var_grid.false_easting = cs.false_easting
+                    cf_var_grid.false_northing = cs.false_northing
+                    # Only one of these should be set
+                    if cs.true_scale_lat is not None:
+                        cf_var_grid.true_scale_lat = cs.true_scale_lat
+                    elif cs.scale_factor_at_projection_origin is not None:
+                        cf_var_grid.scale_factor_at_projection_origin = (
+                            cs.scale_factor_at_projection_origin
+                        )
+                    else:
+                        cf_var_grid.scale_factor_at_projection_origin = (
+                            1.0  # TODO: Is this right?
```

Comment:
Also this @pp-mo

Reply:
Hmm.  In this case, [the CF spec](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#polar-stereographic) does insist on having _one_ of scale-factor / standard-parallel.  
So I guess scale-factor is easiest, and 1.0 make some kind of sense.